src/compiler/diagnosticMessages.json

```diff
@@ -5023,11 +5023,11 @@
         "category": "Error",
         "code": 17015
     },
-    "JSX fragment is not supported when using --jsxFactory": {
+    "The 'jsxFragmentFactory' compiler option must be provided to use JSX fragments with the 'jsxFactory' compiler option.": {
         "category": "Error",
         "code": 17016
     },
-    "JSX fragment is not supported when using an inline JSX factory pragma": {
+    "An @jsxFrag pragma is required when using an @jsx pragma with JSX fragments.": {

```

Comment:
Hopefully this doesn’t turn into the GIF pronunciation debate, but I don’t pronounce the `@` character so I think these articles should be “a,” not “an” 🙃

Reply:
Pretty glad we moved away from "working @﻿microsoft" to "working at @﻿microsoft"

Reply:
... I pronounce the `@`. 😆

Reply:
When @weswigham first mentioned this as PR comment, I didn't fully agree with him, but when I read it over many times I was internally conflicted which way is the right way. In the end I chose to appease @weswigham 🤷